recipe/setup_compiler.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+if [[ ! -d  $SRC_DIR/cf-compilers ]]; then
+    conda create -p $SRC_DIR/cf-compilers -c conda-forge/label/sysroot-with-crypt -c conda-forge --yes --quiet \
```

Comment:
So I tried to look for where this label got built, but AFAICT it just got added after the fact to a package in `main`, i.e. the last builds before https://github.com/conda-forge/linux-sysroot-feedstock/pull/54 / https://github.com/conda-forge/linux-sysroot-feedstock/pull/55. 

I saw you already [answered](https://github.com/conda-forge/ctng-compilers-feedstock/pull/127#discussion_r1573853024) a question about this in the last PR, but leaving this as a review note for posterity, because it's an unusual situation (i.e. labels normally get created in-feedstock, which isn't the case here)

Reply:
Yes, I added it later. Not sure what else to do.

Reply:
I mean if we want to, we could add branches in `linux-sysroot` corresponding to the state before the crypt-removal, add a `channel_targets` in CBC, and push those branches. That way it'd be easy to discover where the builds come from.

Reply:
Not worth it, since it's not needed in gcc 14 forwards.

Reply:
Fine by me. Could you leave a comment that we can remove the extra channel once we have GCC compilers?